src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,120 @@
+package duke.storage;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class for the representation of the storage/database.
+ */
+public class Storage {
+
+    private String databasePath;
+
+    public Storage(String databasePath) {
+        this.databasePath = databasePath;
+    }
+
+    /**
+     * Real all the tasks from the database.
+     * @return the list of all the tasks from the database
+     */
+    public List<Task> readAll() {
```

Comment:
Method name might be a little vague? Since the method returns a list.

Reply:
Hmm, yeah, I kind of follow the idea of CRUD (the read), as this method scans through the database, therefore I name it readAll. Any suggestions?